tools/travis/stage.sh

```diff
@@ -212,6 +219,9 @@ function get_test_modules_for_stage() {
         (${STAGE_TESTS})
             echo "-pl $modules_tests"
         ;;
+        (${STAGE_CONNECTOR_HIVE_1})
+            echo "-pl $MODULES_CONNECTOR_HIVE -Phive-1.2.1"
```

Comment:
on a side-note, if you would include clean here you'd only recompile the hive module, but it's quite a hack.

Reply:
This sounds like a good hack to me. Do you think we can do it?